src/coreclr/pal/src/map/virtual.cpp

```diff
@@ -1057,8 +1057,11 @@ VirtualFree(
         }
 
         TRACE( "Un-committing the following page(s) %d to %d.\n",
```

Comment:
It looks like this uses incorrect formatting characters (preexisting to your change), the arguments are memory addresses, which can be 64 bit.